addons/core/translations/resources/en-us.yaml

```diff
@@ -354,3 +354,6 @@ scope:
 managed-group:
   title: Managed Group
   title_plural: Managed Groups
+  description: A managed group is a resource that represents a collection of accounts.
```

Comment:
"A managed group represents a collection of accounts."  I'd also mention something about the automated/managed nature via a filter.  BE might be able to help here.

Reply:
- Change done.
- Also, asked BE some help with enhancing this description.